tutorials/tutorial01_sumo.ipynb

```diff
@@ -353,9 +353,9 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Round 0, return: 842.099237721961\n",
-      "Average, std return: 842.099237721961, 0.0\n",
-      "Average, std speed: 3.0266213709996355, 0.0\n",
+      "Round 0, return: 776.1625031209684\n",
```

Comment:
I think it would be good to remove the output from this tutorial

Reply:
done!